### ISSUE TITLE
[BUG] Visual Studio Code Does not identify the correct version set as global, nor the version identified in the project

### ISSUE THREAD

**Issue by claudneysessa** [leoafarias/fvm#585]
Hello,

I'm working on a Macbook Pro M1, I use Visual Studio Code, and I identified that the vs code inside the project folder behaves in a very different way.

<img width="1440" alt="Captura de Tela 2023-12-16 às 21 48 44" src="https://github.com/leoafarias/fvm/assets/12506432/4207b6f3-e6b3-4806-8581-9e15d211ec4f">

Note that I have version `3.16.2` set as `global`, it is set as the project version with `fvm use` and even so flutter --version still points to `3.5.7`

I'm having a problem that I haven't been able to identify, I've had FVM installed correctly for years, this has never happened before, I need help to resolve it.

Below is the details of the procedures I have already carried out:

```bash
  fvm doctor
  flutter pub cache repair
```

Log do fvm doctor

```dart
➜  fvm_bug_report_3_7_5 flutter --version

Flutter 3.7.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision c07f788888 (10 months ago) • 2023-02-22 17:52:33 -0600
Engine • revision 0f359063c4
Tools • Dart 2.19.2 • DevTools 2.20.1

➜  fvm_bug_report_3_7_5 fvm list         

Cache Directory:  /Users/claudneysessa/fvm/versions

stable
3.16.2 (active) (global)
3.7.5
1.22.0

➜  fvm_bug_report_3_7_5 fvm doctor       

FVM Version: 2.4.1
___________________________________________________

FVM config found:
___________________________________________________

Project: fvm_bug_report_3_7_5
Directory: /Users/claudneysessa/Workspace/github/pessoal/fvm_bug_report_3_7_5
Version: 3.16.2
Project Flavor: None selected
___________________________________________________

Version is currently cached locally.

Cache Path: /Users/claudneysessa/fvm/versions/3.16.2
Channel: false
SDK Version: 3.16.2

IDE Links
VSCode: .fvm/flutter_sdk
Android Studio: /Users/claudneysessa/Workspace/github/pessoal/fvm_bug_report_3_7_5/.fvm/flutter_sdk


Configured env paths:
___________________________________________________

Flutter:
/Users/claudneysessa/fvm/versions/3.7.5/bin/flutter

Dart:
/Users/claudneysessa/fvm/versions/3.7.5/bin/dart

FVM_HOME:
/Users/claudneysessa/fvm

➜  fvm_bug_report_3_7_5 flutter --version

Flutter 3.7.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision c07f788888 (10 months ago) • 2023-02-22 17:52:33 -0600
Engine • revision 0f359063c4
Tools • Dart 2.19.2 • DevTools 2.20.1

➜  fvm_bug_report_3_7_5 
```

Why is it that when using `flutter --version` it does not get the value of the correct version reported as global in `FVM`?

**Comment by mvatecna** [comment]
I have the same issue, with also on Macbook M1

**Comment by leoafarias** [comment]
@claudneysessa @mvatecna  Please use version 3.0.0 beta, and see if the issue persists, as we are releasing it this week

**Comment by leoafarias** [comment]
Closed as we are releasing 3.0. Please reopen after
